Install Codex skills via ~/.agents (#1)
brian-bell/eddy: README.md

4. **Open the vault in Obsidian** — open the `notes/` folder as an Obsidian vault to browse the knowledge graph.

> 5. **Start your coding agent** in the repo folder and you're ready to go:

Author: ```suggestion
5. **Start Claude Code or Codex** in the repo folder and you're ready to go:
```
